<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>469</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>469</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>График</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Построить график</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCustomPlot" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>572</width>
       <height>363</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
